tracee-ebpf/tracee/tracee.go

```diff
@@ -1390,8 +1390,9 @@ func (t *Tracee) updateProfile(sourceFilePath string, executionTs uint64) {
 
 func (t *Tracee) updateFileSHA() {
 	for k, v := range t.profiledFiles {
-		pathPrefix := strings.Split(k, ".")[0]
-		exeName := strings.Split(strings.Split(k, ".")[1], ":")[0]
+		s := strings.Split(k, ".")
+		pathPrefix := s[0]
```

Comment:
It may get optimized out anyway, but we don't need to even create 'pathPrefix', we can just use 's[0]'.